scripts/mint.ts

```diff
@@ -0,0 +1,28 @@
+/*
+  You can use this script to quickly manually mintNFTs. To do so:
+  1. Run `tsc` to compile typescript into js
+  2. Build and deploy the NFTCollection smart contract
+  3. Run `truffle exec ./ts-out/scripts/mint.ts`
+ */
+var NFTCollection = artifacts.require("./NFTCollection.sol");
+
+function getErrorMessage(error: unknown) {
+  if (error instanceof Error) return error.message
+  return String(error)
+}
+
+const main = async (cb: () => void) => {
+  try {
+    const args = process.argv.slice(4);
+    const numNfts = args.length != 0 ? parseInt(args[0]) : 1;
```

Comment:
Is there a way that we can allow the user to decide what `numNfts` is? That way they can mint however many they want. I also may just be confused on the function here and if may already be doing that so if so let me know.

Reply:
that's what this is doing

So they include an argument, but if it doesn't exist, it defaults to 1